packages/runtime/src/useCases/transport/relationshipTemplates/LoadPeerRelationshipTemplate.ts

```diff
@@ -55,7 +55,7 @@ export class LoadPeerRelationshipTemplateUseCase extends UseCase<LoadPeerRelatio
     }
 
     private async loadRelationshipTemplateFromTokenReference(tokenReference: string, password?: string): Promise<Result<RelationshipTemplateDTO>> {
-        const token = await this.tokenController.loadPeerTokenByTruncated(tokenReference, true);
+        const token = await this.tokenController.loadPeerTokenByTruncated(tokenReference, true, password);
```

Comment:
please write a test for this, it's a pity that you didn't discovered this in the other PR by writing proper tests.